dev-libs/granite/granite-5.2.4.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+VALA_MIN_API_VERSION=0.40
+VALA_USE_DEPEND="valadoc"
```

Comment:
This didnt work like we wanted, take it out and put it back to BDEPEND behind doc? ( .. )